osu.Game/Skinning/Skin.cs

```diff
@@ -239,5 +246,50 @@ protected virtual void Dispose(bool isDisposing)
         }
 
         #endregion
+
+        public override string ToString() => $"{GetType().ReadableName()}{{ Name: {Name} }}";
+
+        private static readonly ThreadLocal<int> nested_level = new ThreadLocal<int>(() => 0);
+
+        [Conditional("SKIN_LOOKUP_DEBUG")]
+        internal static void LogLookupDebug(object callingClass, object lookup, LookupDebugType type, [CallerMemberName] string callerMethod = "")
+        {
+            string icon = string.Empty;
+            int level = nested_level.Value;
+
+            switch (type)
+            {
+                case LookupDebugType.Hit:
+                    icon = "🟢";
+                    break;
+
+                case LookupDebugType.Miss:
+                    icon = "🔴";
+                    break;
```

Comment:
Can we have these include text, too? Such as

```suggestion
                case LookupDebugType.Hit:
                    icon = "🟢 hit";
                    break;

                case LookupDebugType.Miss:
                    icon = "🔴 miss";
                    break;
```

Ask because (a) the icons don't show in debug log overlay, and (b) my current font(s) do not seem to support this glyph, instead resulting in a very helpful barrage of log messages which you don't really know what they are,